Duplicati.Library.RestAPI/Database/Connection.cs

```diff
@@ -36,6 +37,7 @@ public class Connection : IDisposable
         public const int ANY_BACKUP_ID = -1;
         public const int SERVER_SETTINGS_ID = -2;
         private readonly Dictionary<string, Backup> m_temporaryBackups = new Dictionary<string, Backup>();
+        private static string[] _encryptedFields = { ServerSettings.CONST.JWT_CONFIG, ServerSettings.CONST.PASSPHRASE_FIELDNAME };
```

Comment:
I was searching for `ArgumentType.Password` in the codebase and found quite a lot of unexpected strings.
Should we look for these dynamically (so we pick up new entries automatically) or compile a list of known password arguments? For instance `auth-password` was used in some places.